core/src/test/java/org/infinispan/jmx/MvccLockManagerMBeanTest.java

```diff
@@ -79,7 +83,12 @@ public void testNumberOfLocksAvailable() throws Exception {
 
       assertAttributeValue("NumberOfLocksHeld", 1);
       assertAttributeValue("NumberOfLocksAvailable", initialAvailable - 1);
-      tm.getTransaction().runCommit(true);
+      try {
+         tm.getTransaction().runCommit(true);
+         fail("RollbackException expected!");
```

Comment:
I would lean towards the 2nd alternative, I don't like the idea that the normal flow of the method is to throw an exception, or having to check the exception message.

OTOH perhaps most tests that use `runCommit(true)` don't really need it, and they can use `rollback()` instead.